getting-started/eclipselink/docker-compose-bootstrap-db.yml

```diff
@@ -20,7 +20,7 @@
 services:
   polaris-bootstrap:
     # IMPORTANT: the image MUST contain the Postgres JDBC driver and EclipseLink dependencies, see README for instructions
-    image: apache/polaris-admin-tool:postgres-latest
+    image: apache/polaris-admin-tool:1.1.0-incubating-SNAPSHOT
```

Comment:
Shouldn't we use `latest` here? Using a specific version here means that we'd need to update it during or immediately after the release process.